language/en_us/admin_domains.php

```diff
@@ -273,7 +273,7 @@
 $lang['AdminDomains.import.text_terms_notice'] = 'At least one term must be selected.';
 
 
-$lang['AdminDomains.getDays.same_day'] = 'Same Day';
+$lang['AdminDomains.getDays.never'] = 'Never';
```

Comment:
This doesn't make sense in either context it's used.  "Renew Days Before Expiration" - "Never" doesn't make sense, it needs to be "same day" as it was.  "Sync Every" - "Never" also doesn't make any sense.  Let's just filter out the 0 day option for the "Sync Every" list